pkg/share/manila/sharebackends/sharebackend_test.go

```diff
@@ -72,6 +72,7 @@ func TestCSICephFSBuildSource(t *testing.T) {
 	}
 
 	args.Options.CSICEPHFSdriver = "csi-cephfs"
+	args.Options.CSICEPHFSmounter = "fuse"
```

Comment:
This can also be `kernel`, right?
Please add a test to keep us from falling back.

Reply:
@dixudx addressed your comment